docs/source/user_guide/model_understanding.ipynb

```diff
@@ -565,6 +565,84 @@
     "                                        include_shap_values=True, output_format=\"dataframe\")\n",
     "report"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Force Plots\n",
+    "Force plots can be generated to predict single or multiple rows for binary, multiclass and regression problem types.  Here's an example of predicting a single row on a binary classification dataset.  The force plots show the predictive power of each of the features in making the negative (\"Class: 0\") prediction and the positive (\"Class: 1\") prediction. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import shap\n",
+    "\n",
+    "from evalml.demos import load_breast_cancer\n",
+    "from evalml.model_understanding.force_plots import graph_force_plot\n",
+    "from evalml.pipelines import BinaryClassificationPipeline\n",
+    "\n",
+    "X, y = load_breast_cancer()\n",
+    "\n",
+    "\n",
+    "pipeline = BinaryClassificationPipeline(component_graph = ['Simple Imputer', 'Random Forest Classifier'])\n",
+    "pipeline.fit(X, y)\n",
+    "\n",
+    "shap.initjs() # Allows plots to be visualized in Jupyter notebook\n",
```

Comment:
Is there a reason why our code shouldn't automate this?

We have a method called `jupyter_check` which should tell you whether or not code is being run in an ipython environment. If true, could `graph_force_plot` call `shap.initjs()` internally?

Relatedly, so far we've only used `jupyter_check` to conditionally import `ipywidgets`, which I believe we do for similar reasons to this call to `shap.initjs()`... would doing that in `graph_force_plot` make this simpler?

Here's an example: https://github.com/alteryx/evalml/blob/main/evalml/model_understanding/graphs.py#L909

Also, I took a look at [what `initjs` is doing](https://github.com/slundberg/shap/blob/master/shap/plots/_force.py#L244). It looks like it checks that ipython is set up, imports the shap logo and displays the logo in an HTML div alongside a javascript bundle.

Reply:
Done and done!